manifest.json

```diff
@@ -15,16 +15,26 @@
     "default_popup": "popup.html"
   },
   "background": {
-    "scripts": ["js/background.js"]
+    "scripts": [
+      "js/browser-polyfill.min.js",
+      "js/background.js"
+    ]
   },
+  "content_scripts": [
+    {
+      "matches": ["*://localhost/*", "https://loadster.app/*", "https://speedway.app/*"],
+      "js": [
+        "js/browser-polyfill.min.js",
+        "js/content.js"
+      ],
+      "run_at": "document_start"
```

Comment:
The `document_start` here refers to when the extension is loaded, right? Not when each page is loaded in the browser?

Reply:
Never mind that question – took another look and now I think I get it. :)